### DISCUSSION TITLE
Add support for gif to the Image Converter

### DISCUSSION THREAD

**Discussion by joergrosenhagen** [DevToys-app/DevToys#360] [Ideas]
I would like to see support for other formats such as gif implemented in the Image Converter. Other perhaps modern formats such as heic would also be conceivable. Or are there licensing problems?

**Comment by rlm96** [comment]
@veler adding that support should be easy, we only have to implement those codecs we removed in the PR, thoughts?

**Reply by Osmiridium** [reply]
It would be great if DevToys users can one day have more options to choose from when selecting the particular combination of digital image file data encoding and compression standards formats to convert their image files to when using the DevToys Image Converter sub-tool in the Windows 10/11 x64 UWP app.

After adding more types of image file data encoding/compression standards (image file formats) to the list of the Image Converter's format/codec conversion capabilities, like GIF, WebP, WebM, JP2/J2K, JXL, JXR, AVIF, AVIC, ALIF, ALIC, FLIF, FLIC, EXR, HDR, TGA, SVG, TIFF/TIF, RGB, BMP, MKV, APNG, MPNG, ANG, MNG, JNG, MJPG, DNG, ICO, ICNS, etc... And there should also be a new sub-menu somewhere in the Image Converter utility's DevToys GUI sub-page which when toggled-open after clicking a button and would then expose a sub-menu list of image file format encoder/decoder/converter compression and encoder quality custom configuration parameter variables users can tune to their preferred encoding/compression convert values of the customizable parameters dictating the visual quality/size/encoding technique and other basic custom parameters advanced users depend on to do their work quickly.

Thanks and keep up the great work, I hope to contribute some code at some point in the future when I have the free time.

**Reply by nickdevore** [reply]
Yes, WebP and HEIC (mainly coming from that to others) would be great

**Comment by Eves101** [comment]
Would be nice to have an option to make the background transparent when converting to gif.
